backend/model/childModel.js

```diff
@@ -0,0 +1,25 @@
+const mongooose = require("mongoose");
+const { Schema } = require("mongoose");
+const AddressSchema = require("./addressSchema")
+
+const ChildSchema = new Schema({
+    firstName: { type: String, required: true },
+    lastName: { type: String, required: true },
+    birthday: { type: Date, min: "1950–01–01", required: true },
+    address: {
+        type: AddressSchema,
+        required: true
+    },
+    groupName: { type: String, required: true },
+    img: { type: String, required: true },
+    allergies: { type: String, required: true },
```

Comment:
allergies:{type:[{to:String}],required:true } ??
dietaryNeeds:{type:[{requirement:String}],required:true } ??